### PR TITLE
Added tests to automatically benchmark schematic APIs

### PR DESCRIPTION
**Pull request by linglp**
Related to the Epic issue here: https://sagebionetworks.jira.com/browse/DCA-105
Related to the Jira issue here: https://sagebionetworks.jira.com/browse/DCA-110

The goal of this PR is to automatically benchmark API endpoints. Highlights are: 
* Created test for generating a new manifest using the example data model as a google sheet 
* Created test for generating a new manifest using the example data model as an excel spreadsheet
* Created test for generating a new manifest using the HTAN data model as a google sheet
* Created test for generating an existing manifest as a google sheet 
* Uploaded the result to synapse as a table: https://www.synapse.org/#!Synapse:syn51365205/tables/

### CODE REVIEW

**Comment by brucehoff** [comment]
@linglp Please name your feature branch according to the Jira issue (e.g., "DCA-xxxx") and also put that ID at the beginning of the descrption.  Doing so will create a link from the Jira issue to the PR.

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          python3.10 -m venv venv
+          source venv/bin/activate
+
+      - name: Install dependencies
+        run: pip3 install -r requirements.txt
+
+      - name: Generate manifest test
+        env:
+          SYNAPSE_ACCESS_TOKEN: ${{ secrets.SYNAPSE_ACCESS_TOKEN }}
+          SERVICE_ACCOUNT_CREDS: ${{ secrets.SERVICE_ACCOUNT_CREDS }}
```
Comment (thomasyu888) [inline .github/workflows/workflow.yml:35]: The synapse python client actually automatically detects the `SYNAPSE_AUTH_TOKEN` environment variable.  I would rename the `SYNAPSE_ACCESS_TOKEN` and remove `SERVICE_ACCOUNT_CREDS`

File: APITests/utils.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    Update a dataframe to synapse as a csv file
+    Args:
+    df: dataframe to upload
+    syn: synapse obj
+    """
+    # for debugging github action
+    return_time_now("Upload result to synapse")
+
+    # save dataframe to a csv
+    df.to_csv("latency.csv", index=False, header=True)
```
Comment (linglp) [inline APITests/utils.py:191]: save the latency.csv as a table?

File: .github/workflows/workflow.yml (hunk 1 above)
Comment (linglp) [reply .github/workflows/workflow.yml]: Thanks Tom. I have updated the code to use SYNAPSE_AUTH_TOKEN

**Pull request merged**